package.json

```diff
@@ -58,7 +58,6 @@
     "once": "^1.4.0",
```

Comment:
I wish I could comment further up this file, but you should bump both the `index` and `content` versions to `2` near the top here, to signal a new cache format.